### ISSUE TITLE
Detect broken MR-JAR files

### ISSUE THREAD

**Issue by sormuras** [sandermak/modulescanner#14]
For example, when a MR-JAR contains a version `9` directory within its `META-INF` facility, it shouldn't contain classes that are compiled with `--release 10` or higher.

See this _broken_ Java module as a reference: https://repo1.maven.org/maven2/org/lwjgl/lwjgl/3.2.2/lwjgl-3.2.2.jar

In its `META-INF/versions/9/` are classes reporting bytecode level `54.0` (Java 10):

![image](https://user-images.githubusercontent.com/2319838/61242559-4c3b1300-a746-11e9-8ac1-431c89d88c03.png)

**Comment by Spasi** [comment]
Correct, the Ant [jar task](https://ant.apache.org/manual/Tasks/jar.html) was used.

**Comment by sormuras** [comment]
Just uploaded a simple "multi-release JAR" sanity checker tool. Find it here: https://github.com/sormuras/modules/blob/master/src/MultiReleaseCheck.java

- `java MultiReleaseCheck.java org.lwjgl:lwjgl:3.1.6` yields `0`, as it isn't a MRJAR
- `java MultiReleaseCheck.java org.lwjgl:lwjgl:3.2.0` yields `1` due to
`  Expected 9, but 'META-INF/versions/9/module-info.class' reports: 10 (54)`

**Comment by sormuras** [comment]
With Java 17, `jar` supports a `--validate` option -- see https://github.com/openjdk/jdk/pull/3971 for details.
